integration-tests/src/endpoints/feeds.py

```diff
@@ -0,0 +1,163 @@
+        ), f"Expected 200 status code, got {response.status_code}."
+        feeds = response.json()
+        assert isinstance(feeds, list), "Response should be a list."
+        assert len(feeds) == 10, f"Expected 10 feeds, got {len(feeds)}."
+
+    def test_feeds_offset(self):
+        """Test feed retrieval with offset"""
+        limit = 10
+        response = self.get_response("v1/feeds", {"limit": limit})
+        assert (
+            response.status_code == 200
+        ), f"Expected 200 status code for initial request, got {response.status_code}."
+        initial_feeds = response.json()
+        assert isinstance(initial_feeds, list), "Initial response should be a list."
+        assert (
+            len(initial_feeds) == limit
+        ), f"Expected {limit} feeds initially, got {len(initial_feeds)}."
+
+        prev_feeds_ids = [feed["id"] for feed in initial_feeds]
+        response = self.get_response("v1/feeds", {"offset": limit})
+        assert response.status_code == 200, (
+            f"Expected 200 status code for offset request, got "
+            f"{response.status_code}."
+        )
+        offset_feeds = response.json()
+        assert isinstance(offset_feeds, list), "Offset response should be a list."
+        for feed in offset_feeds:
+            assert (
+                feed["id"] not in prev_feeds_ids
+            ), f"Feed ID '{feed['id']}' should not appear in previous set."
+
+    def test_feeds_with_status(self):
+        """Test feed retrieval by status"""
+        for status in ["active", "inactive", "deprecated", "development"]:
+            response = self.get_response("v1/feeds", params={"status": status})
+            assert response.status_code == 200, (
+                f"Expected 200 status code for status '{status}', got "
+                f"{response.status_code}."
+            )
+            for feed in response.json():
+                assert (
+                    feed["status"] == status
+                ), f"Expected status '{status}', got '{feed['status']}'."
+
+    def test_feeds_status_sorting_descending(self):
+        """Test sorting of feeds on status in descending order"""
+        response = self.get_response("v1/feeds", params={"sort": "-status"})
+        assert (
+            response.status_code == 200
+        ), f"Expected 200 status code for sorted feeds, got {response.status_code}."
+        feeds = response.json()
+        assert len(feeds) > 1, "Expected more than one feed for sorting test."
+        prev_feed_status = feeds[0]["status"]
+        for feed in feeds[1:]:
+            current_feed_status = feed["status"]
+            assert current_feed_status <= prev_feed_status, (
+                f"Expected feed status to be in descending order, but found '{prev_feed_status}' followed by "
+                f"'{current_feed_status}'."
+            )
+            prev_feed_status = current_feed_status
+
+    def test_feeds_status_sorting_ascending(self):
```

Comment:
`sort` is slated to be removed.
It's replaced by `order_by` that can order by external_id and country_code.

Reply:
I would remove `sort` and add tests for `order_by`